packages/ocamlbrowser/ocamlbrowser.4.08.0/opam

```diff
@@ -15,7 +15,7 @@ install: [
 remove: [["rm" "-f" "%{bin}%/ocamlbrowser"]]
 depends: [
   "ocaml" {>= "4.08"}
```

Comment:
Why not add `< “4.09}` here if the problem is with 4.09?

Reply:
It's just that I don't know for sure what the problem is, and it is not directly related with this commit.